vnet/db/migrations/0006_filters.rb

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+
+Sequel.migration do
+  up do
+
+    create_table(:filters) do
+      primary_key :id
+      String :uuid, :unique => true, :null => false
+      String :mode, :null => false
+
+      Integer :interface_id, :index => true
+      
+      FalseClass :ingress_passthrough, :null=> false
+      FalseClass :egress_passthrough, :null=>false
+
+      DateTime :created_at, :null =>false
+      DateTime :updated_at, :null =>false
+      DateTime :deleted_at, :index => true
+      Integer :is_deleted, :null =>false
+    end
+    
+    create_table(:filter_statics) do
+      primary_key :id
+      Integer :filter_id, :index => true, :null => false
+
+      Bignum :ipv4_src_address, :null => false
+      Bignum :ipv4_dst_address, :null => false
+      Integer :ipv4_src_prefix, :null => false
+      Integer :ipv4_dst_prefix, :null => false
+      Integer :port_src
+      Integer :port_dst
+      
+      String :protocol, :null => false
+      FalseClass :passthrough, :null => false
```

Comment:
Missing a 'unique' that includes filter_id and every column above except passthrough.